UPGRADE-4.2.md

```diff
@@ -5,3 +5,16 @@ Security
 --------
 
  * Using the `has_role()` function in security expressions is deprecated, use the `is_granted()` function instead.
+ * Passing custom class names to the
+   `Symfony\Component\Security\Core\Authentication\AuthenticationTrustResolver` to define
+   custom anonymous and remember me token classes, is deprecated as of 4.2. To
```

Comment:
sorry, forget one thing: we don't need the "as of 4.2" here as the file is specifically for 4.2.